twatbot.py

```diff
@@ -666,7 +685,13 @@ def start_server(ip, port):
 wl_hash = None
 def load_watchlist():
 	global watchlist, wl_hash
-	wl = [x.rstrip() for x in open(args.watchlist, 'r').readlines() if not x.startswith(';')]
+	wl = []
+	if not os.path.exists('users'): os.makedirs('users')
+	for x in open(args.watchlist, 'r').readlines():
+		if not x.startswith(';'):
+			x = x.rstrip()
+			if not os.path.exists('users/%s' % x.lower()): os.makedirs('users/%s' % x.lower())
```

Comment:
doesn't os.makedirs() work like `mkdir -p` ? if so we can remove the above 'users' mkdir
also we should use paths.get_user here